client/ayon_houdini/api/node_wrap.py

```diff
@@ -0,0 +1,41 @@
+import hou
+
+from ayon_core.pipeline import registered_host
+from ayon_core.pipeline.create import CreateContext
+
+
+def make_publishable(node):
+    # TODO: Can we make this imprinting much faster? Unfortunately
+    #  CreateContext initialization is very slow.
+    host = registered_host()
+    context = CreateContext(host)
+
+    # Apply the instance creation to the node
+    context.create(
+        creator_identifier="io.ayon.creators.houdini.publish",
+        variant="__use_node_name__",
+        pre_create_data={
+            "node": node
+        }
+    )
+
+
+# TODO: Move this choice of automatic 'imprint' to settings so studio can
+#   configure which nodes should get automatically imprinted on creation
+# TODO: Do not import and reload the creator plugin file
+from ayon_houdini.plugins.create import create_generic
```

Comment:
Please, never import plugins.

Reply:
> Please, never import plugins. Shouldn't node_type_product_types from the creator be moved here (or to different pipeline file)?

It should be taken from settings instead - or stored elsewhere.

Reply:
I'm not sure about settings. Those values don't seem like something what TD can fill, they actually look like they are quite pre-determined.

Reply:
They are - until you get into the realm of custom HDAs (custom nodes) a studio develops on their own inside of Houdini (or get shipped with addons/plugins for Houdini and are not native to Houdini). We may not know about ALL nodes from our codebase.

Reply:
So, basically, pipeline admins would need to expand that list as needed e.g. to add their own types or like what krishna did at early stage of this PR https://github.com/ynput/ayon-core/pull/542#issuecomment-2179333623 when he tested `labs::karma::2.0` node that we don't usually support in the current status of the addon, correct?

Reply:
Does moving `node_type_product_types` to settings resolve this?

Reply:
> Does moving `node_type_product_types` to settings resolve this?

Yes - but for now I've shuffled the imports around. But moving it to settings is still very relevant down the line.